devtools/conda-envs/test-env.yaml

```diff
@@ -63,8 +56,12 @@ dependencies:
   - pytest-cov
   - pytest-asyncio
   - pytest-celery
+  - pytest-xdist
```

Comment:
From memory, `pytest-xdist` and `pytest-celery` don't play nice together - but I might be misremembering. But I guess it doesn't hurt much to have xdist in the environment if it helps with local stuff, and it won't be used in CI without further changes.

Reply:
`pytest-xdist` doesn't play well when a number of other kids are at the playground - I was surprised it didn't cause any visible issues in CI. I prefer keeping it in (a couple minutes of CI is convenient and it drastically reduces local testing time) with the intent to remove it the moment it causes any trouble